packages/opc-comment-input/src/opc-comment-input.ts

```diff
@@ -1,12 +1,13 @@
-import { LitElement, html, property, customElement, query, internalProperty } from 'lit-element';
+import { LitElement, html, property, customElement, query } from 'lit-element';
+import { state } from 'lit/decorators.js';
 import style  from './opc-comment-input.scss';
 
 @customElement('opc-comment-input')
 export class CommentInput extends LitElement {
   @property({type: String})
   placeholder = 'Type your comment';
 
-  @internalProperty()
+  @state()
   protected commentText = '';
 
   @query('textarea')
```

Comment:
Uses this from `lit/decorators.js` as well. Don't use `iit-element`

Reply:
```
import {html, css, LitElement} from 'lit';
import {customElement, property} from 'lit/decorators.js';

```

Reply:
yeah, updated the imports.